downloads-generation/cross_validation_class1/GENERATE.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Any chance you'll add a comment to the top of this script (and other .sh's) summarizing what they do?

Reply:
Sure, I can add that. High level descriptions are in the README.md file in each directory but I agree more detail would be good in the script